core/lib/mcxtx_api.h

```diff
@@ -135,7 +135,15 @@
      * all.  We do not need anything more than word alignment for OP_vldm/OP_vstm,
      * and dr_simd_t has no fields larger than 32 bits, so we have no padding.
      */
+#ifdef AARCH64
+    dr_simd_t simd[MCXT_NUM_SIMD_SVE_SLOTS];
+#else
     dr_simd_t simd[MCXT_NUM_SIMD_SLOTS];
```

Comment:
Could we define MCXT_NUM_SIMD_SVE_SLOTS for 32-bit and then not need a split array decl here?

Reply:
We can't for 32 bit because the array sizes will be different and the name `MCXT_NUM_SIMD_SVE_SLOTS` would mislead as SVE is only supported on 64 bit.